### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by Matt-Butler**
### Summary
Documentation and examples for terra-react-svg-loader usage.

@cerner/terra

### CODE REVIEW

File: examples/import/index.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import IconGithub from 'babel-loader!terra-react-svg-loader!./github.svg';
+import IconTerra from 'babel-loader!terra-react-svg-loader!terra-icon/lib/icon/static/deviceAlert.svg';
+
+let App = (
```
Comment (zhongyn) [inline examples/import/index.js]: can it be ```const```?

File: examples/webpack1/package.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  },
+  "dependencies": {
+    "babel-core": "^6.23.1",
+    "babel-loader": "^6.3.1",
+    "babel-plugin-transform-object-rest-spread": "^6.23.0",
+    "babel-preset-es2015": "^6.22.0",
+    "babel-preset-react": "^6.23.0",
+    "react": "^15.4.2",
+    "react-dom": "^15.4.2",
+    "terra-react-svg-loader": "git://github.com/cerner/terra-react-svg-loader.git#master",
```
Comment (zhongyn) [inline examples/webpack1/package.json]: will this loader be part of terra-ui mono repo in the future?

File: examples/import/index.js (hunk 1 above)
Comment (Matt-Butler) [inline examples/import/index.js]: updated to const

File: examples/webpack1/package.json (hunk 2 above)
Comment (Matt-Butler) [inline examples/webpack1/package.json]: It will depend on how we end up versioning the mono repo.

**Comment by zhongyn** [comment]
+1

**Comment by tbiethman** [comment]
+1

**Pull request merged**